From fe31405df76e52607bce9a058c188eb268848dd0 Mon Sep 17 00:00:00 2001
From: Razvan Stefanescu <razvan.stefanescu@freescale.com>
Date: Tue, 7 Jul 2015 16:24:33 +0300
Subject: [PATCH 156/214] dpaa2-ethsw: Prefix driver name with dpaa2-

Keep driver naming in sync with the others.

Signed-off-by: Razvan Stefanescu <razvan.stefanescu@freescale.com>
[Original patch extracted from FSL LS2085 SDK EAR5.0,
Layerscape2-SDK-SOURCE-20150828-yocto.iso]
Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 drivers/staging/fsl-dpaa2/ethsw/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/ethsw/Makefile b/drivers/staging/fsl-dpaa2/ethsw/Makefile
index a1132ca..20eb3ac 100644
--- a/drivers/staging/fsl-dpaa2/ethsw/Makefile
+++ b/drivers/staging/fsl-dpaa2/ethsw/Makefile
@@ -1,7 +1,7 @@
 
-obj-$(CONFIG_FSL_DPAA2_ETHSW) += ethsw.o
+obj-$(CONFIG_FSL_DPAA2_ETHSW) += dpaa2-ethsw.o
 
-ethsw-objs := switch.o dpsw.o
+dpaa2-ethsw-objs := switch.o dpsw.o
 
 all:
 	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) modules
-- 
2.5.3

